src/components/Core/ImdbButton/style.js

```diff
@@ -0,0 +1,15 @@
+import { Button } from "@mui/material";
+import { styled } from "@mui/system";
+
+export const StyledButton = styled(Button)(({ theme, isYellow, padding }) => ({
```

Comment:
instead of using the toggle "isYellow", let the user to pass the color through the props, to make sure that the color is been determined do this : 
color: color|| "#5799ef"